pkgs/applications/misc/sweethome3d/default.nix

```diff
@@ -52,6 +52,7 @@ let
       cp "${sweethome3dItem}/share/applications/"* $out/share/applications
 
       makeWrapper ${jre}/bin/java $out/bin/$exec \
+        --set MESA_GL_VERSION_OVERRIDE 2.1 \
```

Comment:
Please add a comment that explains why this is necessary / what goes wrong when you leave it out. That way whoever stumbles on this in the future will know when it is okay to remove it.